tensorflow/core/util/mkl_util.h

```diff
@@ -991,6 +1290,33 @@ inline padding_kind TFPaddingToMklDnnPadding(Padding pad) {
   return padding_kind::zero;
 }
 
+#ifdef ENABLE_MKLDNN_V1
+/// Helper function to create memory descriptor in Blocked format
+///
+/// @input: Tensor dimensions
+/// @input: strides corresponding to dimensions. One can use utility
+///         function such as CalculateTFStrides to compute strides
+///         for given dimensions.
+/// @return: memory::desc object corresponding to blocked memory format
+///          for given dimensions and strides.
+inline memory::desc CreateBlockedMemDescHelper(const memory::dims& dim,
+                                               const memory::dims& strides,
+                                               memory::data_type dtype) {
+  DCHECK_EQ(dim.size(), strides.size());
+  mkldnn_dim_t input_dims[dim.size()];
```

Comment:
The `ENABLE_MKLDNN_V1` guard could be just from this line to line 1315.

Reply:
Done